BootloaderCommonPkg/Library/IppCryptoLib/auth/Ia32/pcpsha512w7as.nasm

```diff
@@ -274,7 +274,9 @@ ASM_PFX(UpdateSHA512W7):
 
    sub      esp,stackSize        ; allocate local buffer (probably unaligned)
    mov      ecx,esp
-   and      esp,-16              ; 16-byte aligned stack
+   ;and      esp,-16             ; 16-byte aligned stack
+   sub      esp,32               ; 32-byte aligned stack
+   and      esp, 0xfffffff0

```

Comment:
Can you try this instead of 32-byte alignment? Line#253.
```
%define stackSize sSize*oSize+dSize*oSize+wSize*qSize
->
%define stackSize sSize*oSize+dSize*oSize+wSize*qSize+4
```
This is from https://github.com/intel/ipp-crypto.

Reply:
Yes, adjusting stack size here fixed it as well. Will retain this change.

Reply:
Thanks for verifying and updating!